### PR TITLE
discogan pytorch

### PR DESCRIPTION
**Pull request by nakul-jindal**

### CODE REVIEW

File: generative_models/DCGAN_pytorch/main.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            D_losses.append(lossD.item())
+            
+            # Check how the generator is doing by saving G's output on fixed_noise
+            with torch.no_grad():
+                fake = netG(fixed_noise).detach().cpu()
+            img_list.append(torchvision.utils.make_grid(fake, padding=2, normalize=True))
+   
+
+
+from utils import compare_img , plot_loss , animation
```
Comment (nirmal-suthar) [inline generative_models/DCGAN_pytorch/main.py:105]: move imports to top

**Review by nirmal-suthar — COMMENTED** [review]
remove DCGAN from this PR

**Comment by nakul-jindal** [comment]
> remove DCGAN from this PR

done

File: generative_models/DiscoGAN_pytorch/README.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+$ python3 main.py --style_A
+```
+> **_NOTE:_** on Colab Notebook use following command:
+```python
+!git clone link-to-repo
+%run main.py --style_A
+```
+
+## Help log
+```bash
```
Comment (nirmal-suthar) [inline generative_models/DiscoGAN_pytorch/README.md:13]:
```suggestion
```
```

File: generative_models/DiscoGAN_pytorch/model.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+                nn.BatchNorm2d(64 * 8),
+                nn.LeakyReLU(0.2, inplace=True),
+
+                nn.ConvTranspose2d(64 * 8, 64 * 4, 4, 2, 1, bias=False),
+                nn.BatchNorm2d(64 * 4),
+                nn.ReLU(True),
+                nn.ConvTranspose2d(64 * 4, 64 * 2, 4, 2, 1, bias=False),
+                nn.BatchNorm2d(64 * 2),
+                nn.ReLU(True),
+                nn.ConvTranspose2d(64 * 2,     64, 4, 2, 1, bias=False),
```
Comment (nirmal-suthar) [inline generative_models/DiscoGAN_pytorch/model.py]:
```suggestion
                nn.ConvTranspose2d(64 * 2, 64, 4, 2, 1, bias=False),
```

File: generative_models/DiscoGAN_pytorch/model.py [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+                nn.BatchNorm2d(64),
+                nn.ReLU(True),
+                nn.ConvTranspose2d(    64,      3, 4, 2, 1, bias=False),
```
Comment (nirmal-suthar) [inline generative_models/DiscoGAN_pytorch/model.py]:
```suggestion
                nn.ConvTranspose2d(64, 3, 4, 2, 1, bias=False),
```

Comment (nakul-jindal) [reply generative_models/DiscoGAN_pytorch/model.py]: done all 3 changes
